openstudiocore/src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -138,31 +143,37 @@ namespace openstudio{
       if (m_connectionOpen)
       {
         try {
-          execAndThrowOnError("DROP INDEX rmdTI;");

```

Comment:
@mbadams5 why would you ever call removeIndexes?  I don't think we need to remove this method just curious